core/src/price_finding/price_finder_interface.rs

```diff
@@ -50,6 +50,22 @@ impl InternalOptimizer {
     }
 }
 
+/// The optimization algorithm used by standard and best-ring solvers.
+#[derive(Clone, Copy, Debug)]
+pub enum OptModel {
+    Mip,
```

Comment:
ubernit: given that we are not super familiar with the optimization models and their abbreviations, maybe we can write MixedIntegerProgramming out in the enum definition?

Reply:
That's an awesome suggestion, having to write MIP as Mip gave me a sleepless night already! I went with `OptModel::MixedInteger` which is much more pleasing to my eyes now. Thanks!